src/utils/calculatingUtils.tsx

```diff
@@ -0,0 +1,242 @@
+import { createClient } from 'urql'
+
+export const calculateUSDValue = async(value: number | string | any, token: string) => {
```

Comment:
this function has a lot of duplicated code, and it won't work with new currencies added -- would recommend abstraction & modification to the `chains` repo -- so users can specify how to query the currency's USD value